src/context/auth.js

```diff
@@ -82,8 +111,35 @@ const ProtectedRoute = ({ children }) => {
 			<Navigation />
             <Modal />
 			{children}
+
 		</div>
 	)
 }
+function validationEmail(email) {
+	const errorMessage = "Please enter a valid email"
+	const emptyErrorMessage = "Email is required."
+	if (!email) {
+		return emptyErrorMessage
+	}
+	const pattern = /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/
+	if(!pattern.test(email)) return errorMessage
+	return true
+}
+
+function validationPassword(password) {
+	const minLength = 8
+	const hasUppercase = /[A-Z]/.test(password)
+	const hasNumber = /\d/.test(password)
+	const hasSpecialCharacter = /[!@#$%^&*(),.?":{}|<>]/.test(password)
+
+	if(password.length < 1) return 'Password is required'
+	if(password.length < minLength ) return 'Password should be at least 8 characters long'
+	if(password.length > 24) return 'Password should be less than 24 characters'
```

Comment:
Probably wait for the database changes, if this is required surely could be used, but for right now, i guess this is not supposed to be added.